src/App.tsx

```diff
@@ -1,21 +1,14 @@
 import {
```

Comment:
Unfortunately, I still need some MUI elements for the dropdown. I wasn't able to use a Tailwind CSS dropdown that also vanishes when I click somewhere. Also there I have to import a library that uses Tailwind. But maybe one of you have an idea?

Reply:
We should get rid of MUI. I have ideas for this - let's discuss separately

Reply:
No more MUI.